src/test/resources/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning/varierendeAktivitetsdager/komplisert_case.feature

```diff
@@ -0,0 +1,66 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Beregning - Komplisert scenario
```

Comment:
Se beskrivelse om du vil ha diagram for perioder og markert i kalender hvilke dager som telles